docker-compose.yml

```diff
@@ -99,6 +101,8 @@ services:
       - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
       - KAFKA_DELETE_TOPIC_ENABLE=true
       - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+      # The following option can be used in local, but should not be used in production.
+      # It constrains the initial/maximum virtual memory allocated for the JVM.
```

Comment:
I'm not sure I agree: this option should be used in production with appropriate settings right ? see https://docs.confluent.io/platform/current/kafka/deployment.html#jvm

Reply:
The option can surely be used in production in the right context, with the right value and related set of options.
Using `-Xmx256M` will lead to swaps.